appendices.tex

```diff
@@ -90,105 +90,91 @@ \section{Prelude to Lie Algebroids}
-\begin{theorem}
-  Let $M$ be a smooth manifold. Then $M$ might have a symplectic, almost complex and/or Riemannian structure, denoted by $\omega$, $J$ and $g$ respectively. Given two of these structures, we can use them to construct the remaining one, in a compatible way.
-\end{theorem}
-\begin{proof}
-  The reader may verify that the following indeed defines each of the structures as claimed.
-  \begin{enumerate}
-    \item $\omega=g\circ (J\otimes\text{id})$
-    \item $g=\omega\circ(\text{id}\otimes J)$
-    \item Given $\omega$ and $g$, we obtain $J$ through polar decomposition, see \cite{book:cds}.
-          %https://math.mit.edu/classes/18.157/S2012/lecturesonsymplecticgeometryanacannas.pdf
-  \end{enumerate}
-\end{proof}
 \begin{corollary}
-  Any complex manifold with a metric has a symplectic structure defined by $\omega=g\circ (J\otimes\text{id})$. In particular, any projective complex manifold $Z\subset\mathbb{CP}^n$ has a canonical symplectic structure defined by $\omega(X,Y)=\iota^*h(J(X),Y)$.
+    The $2$-form defined by $\omega=h\circ J\otimes\text{id}$ is closed and non-degenerate. That is, $(\mathbb{CP}^n, \omega)$ is a symplectic manifold.
 \end{corollary}
+Notice that we are dealing with a very special manifold here. It has \textit{two} integrable structures. A complex structure, and a symplectic structure, which are compatible in the sense that $\omega(\cdot, J\cdot)$ defines a metric.
+\begin{definition}
+    A complex symplectic manifold $(M,\omega, J)$ such that $g=\omega(\cdot, J\cdot)$ defines a metric is called a Kähler manifold.
+\end{definition}
+\begin{example}
+    Every complex submanifold $M\subseteq\mathbb{CP}^n$ is a Kähler manifold, by pulling back the Fubini-Study $2$-form to $M$. Notice that it isn't true that an arbitrary smooth submanifold of $\mathbb{CP}^n$ is a Kähler manifold! The condition that it is a \textit{complex} submanifold is crucial. Kähler manifolds are of great importance in algebraic geometry, and are also used by string theorists. 
+\end{example}
 Next, we have an example from the realm of Poisson geometry. This is in some sense a generalisation of symplectic geometry. The theory of Poisson manifolds is a very active research area, related to quantisation of physical theories, as well as modelling constraints on physical systems.
 \begin{definition}
-  An almost Poisson manifold is a pair $(M,\Pi)$ such that the bracket $\{\cdot,\cdot\}:C^\infty(M)\times C^\infty(M)\to C^\infty(M)$ defined by $\{f,g\}=\Pi(df,dg)$ satisfies the axioms of a Lie algebra, save for the Jacobi identity. If the bracket satisfies the Jacobi identity, the pair is called a Poisson manifold.
+  An almost Poisson manifold is a pair $(M,\pi)$ such that the bracket $\{\cdot,\cdot\}:C^\infty(M)\times C^\infty(M)\to C^\infty(M)$ defined by $\{f,g\}=\pi(df,dg)$ satisfies the axioms of a Lie algebra, save for the Jacobi identity. If the bracket satisfies the Jacobi identity, the pair is called a Poisson manifold.
```

Comment:
```suggestion
  An \emph{almost Poisson manifold} is a pair $(M,\pi)$ such that the bracket $\{\cdot,\cdot\}:C^\infty(M)\times C^\infty(M)\to C^\infty(M)$ defined by $\{f,g\}=\pi(df,dg)$ satisfies the axioms of a Lie algebra, save for the Jacobi identity. If the bracket satisfies the Jacobi identity, the pair is called a Poisson manifold.
```

Reply:
I liked $\Pi$ more, we used $\pi$ usually for projections

Reply:
I'd like to cite the recent book by Krainic and Marcut here, for readers that would like to know more.